File: Taller-5/Taller5/src/taller5/meses.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package taller5;
import java.util.Scanner;
/**
 *
 * @author SALAA
 */
public class meses {
    public static void main(String[] args) {
        Scanner leer = new Scanner(System.in);
        
        //Declaración e Inicialización de Variables
        int meses;
        
        //Ingreso de los Datos
        System.out.println("Identificador de Meses \n");
        
        System.out.println("Ingrese el número del mes a identificar: ");
        meses = leer.nextInt();
        
        //Proceso y Salida
        
        switch (meses){
            case 1: {
                System.out.println("El número del mes ingresado: " +meses +" corresponde a enero");
            }
            break;
            case 2: {
                System.out.println("El número del mes ingresado: " +meses +" corresponde a febrero");
            }
            break;
            case 3: {
                System.out.println("El número del mes ingresado: " +meses +" corresponde a marzo");
            }
            break;
            case 4: {
                System.out.println("El número del mes ingresado: " +meses +" corresponde a abril");
            }
            break;
            case 5: {
                System.out.println("El número del mes ingresado: " +meses +" corresponde a mayo");
            }
            break;
            case 6: {
                System.out.println("El número del mes ingresado: " +meses +" corresponde a junio");
            }
            break;
            case 7: {
                System.out.println("El número del mes ingresado: " +meses +" corresponde a julio");
            }
            break;
            case 8: {
                System.out.println("El número del mes ingresado: " +meses +" corresponde a agosto");
            }
            break;
            case 9: {
                System.out.println("El número del mes ingresado: " +meses +" corresponde a septiembre");
            }
            break;
            case 10: {
                System.out.println("El número del mes ingresado: " +meses +" corresponde a octubre");
            }
            break;
            case 11: {
                System.out.println("El número del mes ingresado: " +meses +" corresponde a noviembre");
            }
            break;
            case 12: {
                System.out.println("El número del mes ingresado: " +meses +" corresponde a diciembre");
            }
            break;
            default: 
                System.out.println("El número de mes ingresado es incorrecto");
                break;
        }
        
    }
}

File: Talleres-1B/Taller-5/meses.py
print("\t***Identificador de Meses***\n")

#Switch usando Diccionarios
def switch():
    meses = int(input("Ingrese el número del mes a identificar: "))

    def ene():
        print("El número del mes ingresado: ", meses, " corresponde a enero")
    
    def feb():
        print("El número del mes ingresado: ", meses, " corresponde a febrero")
    
    def mar():
        print("El número del mes ingresado: ", meses, " corresponde a marzo")
    
    def abr():
        print("El número del mes ingresado: ", meses, " corresponde a abril")
    
    def may():
        print("El número del mes ingresado: ", meses, " corresponde a mayo")
    
    def jun():
        print("El número del mes ingresado: ", meses, " corresponde a junio")

    def jul():
        print("El número del mes ingresado: ", meses, " corresponde a julio")

    def ago():
        print("El número del mes ingresado: ", meses, " corresponde a agosto")

    def sep():
        print("El número del mes ingresado: ", meses, " corresponde a septiembre")

    def oct():
        print("El número del mes ingresado: ", meses, " corresponde a octube")

    def nov():
        print("El número del mes ingresado: ", meses, " corresponde a noviembre")

    def dic():
        print("El número del mes ingresado: ", meses, " corresponde a diciembre")
    
    def default():
        print("El número de mes ingresado es incorrecto")
    
    #Mapeo de Diccionario
    dict = {
        1 : ene,
        2 : feb,
        3 : mar,
        4 : abr,
        5 : may,
        6 : jun,
        7 : jul,
        8 : ago,
        9 : sep,
        10 : oct,
        11 : nov,
        12 : dic
    }
    dict.get(meses, default)()
    
switch()